clients/drcachesim/CMakeLists.txt

```diff
@@ -171,6 +171,7 @@ target_link_libraries(drmemtrace_invariant_checker drdecode)
 
```

Comment:
> i#5912: Modifications to PC discontinuity checks

"Modifications" is a little general: how about "Add caching and clean up PC discontinuity checks"?